README.md

```diff
@@ -83,45 +85,50 @@ target 'TargetName' do
+In addition, you should [modify the properties and capabilities of your project's target](https://github.com/getyoti/yoti-doc-scan-ios?tab=readme-ov-file#5-modify-the-properties-and-capabilities-of-your-projects-target).
 
 ## Usage
 ### 1. Import module
 ```javascript
 import RNYotiDocScan from '@getyoti/yoti-doc-scan-react-native';
 ```
-### 2. Launch a session
-Launch a session with its required parameters using the `startSession` function.
+### 2. Optional configuration and customization
+For customization on Android, you can refer to the documentation outlined [here](https://github.com/getyoti/yoti-doc-scan-android?tab=readme-ov-file#customisation). In addition, you can choose to also set a request code:
 ```javascript
-const successCallback = (code, description) => {
-    ...
-}
-const errorCallback = (code, description) => {
-    ...
-}
-RNYotiDocScan.startSession(id, token, successCallback, errorCallback);
+RNYotiDocScan.setRequestCode(0); // default: 9001
 ```
 
-### 3. Customizations
-On iOS, you can set the primary colors using the following API:
+On iOS, the SDK expects launched sessions to contain multiple flows by default. To enable single-flow sessions, configure and include [yoti-doc-scan-react-native-configuration-ios.json](templates/yoti-doc-scan-react-native-configuration-ios.json) in your project’s target and then set the configuration as follows:
 ```javascript
-RNYotiDocScan.setLightPrimaryColorRGB(0, 0, 0); // default: (40, 117, 188)
-RNYotiDocScan.setDarkPrimaryColorRGB(0, 0, 0); // default: (145, 190, 255)
+RNYotiDocScan.setConfiguration({
+    bundle_identifier: "", // Optional: defaults to the main bundle identifier if not specified.
+    filename: "yoti-doc-scan-react-native-configuration-ios.json"
+});
 ```
-To customize the colors on Android, please refer to its separate [documentation](https://github.com/getyoti/yoti-doc-scan-android#colours).
+Ensure that only one module type property is set to `true` when `single_flow` is enabled.
 
-In addition, you can choose to also specify a request code on Android:
+To customize the appearance on iOS, you can configure the SDK using [yoti-doc-scan-react-native-configuration-with-theme-ios.json](templates/yoti-doc-scan-react-native-configuration-with-theme-ios.json) instead, which supports the following options for theming:
+
+- Light and dark mode color themes. We also support specifying only a primary color for each mode
+- Typography theme (system and custom fonts, font weight, size, line height multiple and kern)
+- Spacing mode (compact, regular and relaxed)
+- Shape theme (corner radius and border width)
+- Icon theme (custom vectors, system and custom SF Symbols, incl. localized ones)
+- Illustration theme (custom vectors)
+
+All customisation types are optional, and can be set independently from each other.
+
+### 3. Launch a session
+Launch a session with its required parameters using the `startSession` function.
```

Comment:
We should update the doc as well and mention `start` instead `startSession`

Reply:
Done: 7fabefd27b9abd4c41995f0653ce306f9b9d0b0a